app/assets/javascripts/components/common/OverviewStats/namespace_overview_stats.jsx

```diff
@@ -2,7 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import OverviewStat from './overview_stat';
 
-const NamespaceOverviewStats = ({ statistics }) => {
+const NamespaceOverviewStats = ({ course, statistics }) => {
+  let student_editors_stat_msg;
+  if (course.type === 'ClassroomProgramCourse') {
```

Comment:
You can use CourseUtils.i18n here to handle the if/else behavior. See how it's used in course_alerts.jsx, for example.

Reply:
@ragesoss Thanks Sir, I have updated the code